src/pages/Conjugate/index.jsx

```diff
@@ -28,16 +28,19 @@ const Conjugate = () => {
     value = decodeURI(value).toLowerCase();
     if (value === verb) return;
     setSearchValue(value);
-    setAction('redirect');
+    history.push(`/conjugate/${value}`);
   };
 
   useEffect(() => {
-    // Prevent memory leak by aborting fetch when component unmounts
-    const abortController = new AbortController();
-    const { signal } = abortController;
-
```

Comment:
The abort controller needs to be kept to prevent a memory leak when the component unmounts - please do not remove